app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,69 @@
+class Admin::UsersController < ApplicationController
+  before_action :if_not_admin
+  before_action :set_user, only: [:show, :edit, :destroy, :update]
+
+  def index
+    @users = User.all
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to admin_users_path(@user.id)
+    else
+      render :new
+    end
+  end
+
+  def show
+    @user = User.find(params[:id])
+    # @tasks = @user.tasks
+    @tasks = @user.tasks.select(:task_name)
+  end
+
+  def edit
+  end
```

Comment:
Controllerのアクションにおいて、メソッドの中身がない場合にも、メソッド名と同様のテンプレートをrenderする処理が行われますが、この場合はメソッドを1行で記載できます。

全体のコード量が減るだけでなく、メソッドの中身を書き忘れていないことを明示できるというメリットがあります。
```suggestion
  def edit; end
```